extension/content.js

```diff
@@ -384,7 +389,7 @@ document.addEventListener('DOMContentLoaded', () => {
 				addBlameParentLinks();
 			}
 
-			if (pageDetect.isRepoRoot()) {
+			if (pageDetect.isRepo()) {
 				addReadmeEditButton();
 			}
```

Comment:
You are already in `pageDetect.isRepo()` if, no need for this one. However I think we can add a tree check to `pageDetect` and run this `if(pageDetect.isRepoRoot() || pageDetect.isRepoTree())`.